dot/services.go

```diff
@@ -429,3 +431,8 @@ func newSyncService(cfg *Config, st *state.Service, fg sync.FinalityGadget, veri
 func createDigestHandler(st *state.Service) (*digest.Handler, error) {
 	return digest.NewHandler(st.Block, st.Epoch, st.Grandpa)
 }
+
+func createPprofService() (service *pprof.Service) {
+	pprofLogger := log.New("pkg", "pprof")
+	return pprof.NewService("localhost:6060", pprofLogger)
```

Comment:
i saw you have a todo for making the port configurable, do you want to do that in this PR?

Reply:
Yes I'll 😉 My bad, I wanted to make a draft PR but my finger decided not to somehow. Thanks for the ⚡ review though!